node/src/command.rs

```diff
@@ -146,17 +144,24 @@ pub fn run() -> sc_cli::Result<()> {
 
 						cmd.run::<Block, service::ExecutorDispatch>(config)
 					},
-					BenchmarkCmd::Block(cmd) => cmd.run(client),
+					BenchmarkCmd::Block(cmd) => {
+						let PartialComponents { client, .. } = service::new_partial(&config)?;
```

Comment:
What's the reason for this change?

Reply:
Substrate changed the Benchmarks: https://github.com/paritytech/substrate/blob/563f4820d8f36d256ada7ea3fef46b2e94c4cd5a/bin/node-template/node/src/command.rs#L126

Reply:
I see. Thanks for the info